app/views/prison/visits/_rejection_reason_checkbox.erb

```diff
@@ -0,0 +1,4 @@
+<%= rf.object.checkbox_for(id, id: id, data: {
```

Comment:
I'd be tempted to rename this partial to `_rejection_reason` since it is not only a checkbox. Also, should the extension be `html.erb` ?